Makefile

```diff
@@ -49,3 +49,18 @@ origin-release:
 	@echo "  docker push $(IMAGE_ORG)/origin-release:latest"
 	@echo "  docker push $(IMAGE_ORG)/origin-cluster-kube-apiserver-operator"
 	@echo "  OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=$(IMAGE_ORG)/origin-release:latest bin/openshift-install cluster --log-level=debug"
+
+GO_PACKAGES :=./pkg/... ./cmd/...
+
+GO_E2E_PACKAGES :=./e2e/operator/...
```

Comment:
please use `./test/e2e/operator`

Reply:
As per our discussion, changed to `./test/e2e/...`.